### PR TITLE
added documentation on how to update the documentation

### PR DESCRIPTION
**Pull request by vincentberenz**
[//]: # "Thanks for your contribution.  To make life of the reviewers easier,"
[//]: # "please give this pull request a meaningful title and provide the"
[//]: # "requested information below."

## Description

I added a doc on how to update the doc

### CODE REVIEW

File: E5_documentation.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Developer's guide: updating the documentation
+
+Everybody is welcome to update/extend the documentation.
+
+For doing so:
+
+- clone the documentation [repository](https://github.com/intelligent-soft-robots/pam_documentation)
+- update the documentation
+- If you add a new file: note that there is no need for updating an index or a the list of contents. This is done automatically.
+- If you add a new file: in the content list, the orders of the page is alphabetical. Name your new markdown file based on the order into which it should appear in the content list.
```
Comment (luator) [inline E5_documentation.md]: This is not the case anymore, there is now an `index.rst` that needs to be maintained manually.

Comment (luator) [reply E5_documentation.md]: I would also suggest to use RST for new files, as I think it is better suited for larger documentations (e.g. offers labels that can be referenced and things like "info"/"warning" boxes).  Markdown still works as well, though, so no strict requirement here.

**Review by luator — CHANGES_REQUESTED** [review]
Maybe add a point that once can build locally by simply running `make html`.  Only dependencies are sphinx and the stuff listed in requirements.txt.

File: E5_documentation.md (hunk 1 above)
Comment (vincentberenz) [reply E5_documentation.md]: @luator done !

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,4 @@
 *~
 _build/
+_static/
+_templates/
```
Comment (luator) [inline .gitignore]: What is the reason for ignoring them?  I think right now they are empty (except of a `.keep` file for git) but they are generally part of the documentation source files.

**Review by luator — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (vincentberenz) [reply .gitignore]: I just assumed they were created by make, without looking much into it. I removed them from gitignore

**Review by dtrb — APPROVED** [review]
lgtm

**Pull request merged**